app/frontend/app/controllers/story.js

```diff
@@ -21,6 +21,14 @@ export default Ember.ObjectController.extend(HasCurrentUser, {
   loadingAll: false,
   loadedAll: propertyEqual('substories.length', 'model.substoryCount'),
 
+  // Determine if we are on the stories group page
+  // this is really hacky
+  isOnGroupPage: function() {
+    // 8 == /groups/
+    return this.get('model.group') &&
+      (this.get('target.target.url').substr(8) === this.get('model.group.id'));
```

Comment:
The ApplicationController holds `currentPaht` and `currentRouteName`, that should get this done a bit less hackier.